sapi/phpdbg/phpdbg_prompt.c

```diff
@@ -221,11 +221,11 @@ static void phpdbg_line_init(char *cmd, struct phpdbg_init_state *state) {
 
 	if (*cmd && cmd_len > 0L && cmd[0] != '#') {
 		if (cmd_len == 2) {
-			if (memcmp(cmd, "<:", sizeof("<:")-1) == SUCCESS) {
+			if (memcmp(cmd, "<:", strlen("<:")) == SUCCESS) {
 				state->in_code = 1;
 				return;
 			} else {
-				if (memcmp(cmd, ":>", sizeof(":>")-1) == SUCCESS) {
+				if (memcmp(cmd, ":>", strlen(":>")) == SUCCESS) {
```

Comment:
Note: but these usages of ``SUCCESS`` should probably be removed in favour of using ``0`` explicitly.